src/app/pages/wallets/my-wallets/wallet-form-modal/dialog-create-or-update-wallet/wallet-form-modal.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]='form'>
```

Comment:
Please move all those files from `dialog-create-or-update-wallet` to one level higher, to `wallet-form-modal`